### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by ZorAttC** [DepthAnything/Video-Depth-Anything#27]
Hi, it is such a remarkable work that resolve the video depth estimation problem in a real-time performance.I wanna know if I want to  train the model on my own data e.g. the ViT-s one.How many computation resources should I have ? And Is there any idea for sharing some simple training templates.

**Comment by guohengkai** [comment]
Hi, we have plans to release the training loss.
For computation resources, we use 8xA100(80G) for ViT-large model. For the small one, 8xV100(32G) are enough.
